metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -1133,6 +1135,27 @@ def gen_dataset_workunits(
             project_id=project_id, dataset_name=dataset_name, table=table.name
         )
 
+        if (
+            isinstance(table, BigqueryTable)
+            and table.table_type is not None
+            and table.table_type == "EXTERNAL"
+            and table.ddl is not None
+            and f"CREATE EXTERNAL TABLE `{project_id}.{dataset_name}.{table.name}`"
```

Comment:
hi @mayurinehate 
Apart from this, it will have OPTIONS
for eg 
```
CREATE EXTERNAL TABLE `my-gcp-project-425521.sample_dataset.testing_1`
OPTIONS(
  format="PARQUET",
  uris=["gs://sample_bigquery_data/bq_tables/*.parquet"]
);
```
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#create_external_table_statement